config/cmake/examples/CTestScript.cmake

```diff
@@ -21,17 +21,20 @@ if (NOT SITE_OS_NAME)
   ## machine name not provided - attempt to discover with uname
   ## -- set hostname
   ## --------------------------
-  find_program (HOSTNAME_CMD NAMES hostname)
-  exec_program (${HOSTNAME_CMD} ARGS OUTPUT_VARIABLE HOSTNAME)
-  set (CTEST_SITE  "${HOSTNAME}${CTEST_SITE_EXT}")
+  #find_program (HOSTNAME_CMD NAMES hostname)
+  #execute_process (COMMAND ${HOSTNAME_CMD} OUTPUT_VARIABLE HOSTNAME OUTPUT_STRIP_TRAILING_WHITESPACE)
+  #set (CTEST_SITE  "${HOSTNAME}${CTEST_SITE_EXT}")
```

Comment:
Comment should be removed unless there's a specific reason to keep this around

Reply:
Allen's not sure whether or not this change will be permanent.

Reply:
Updated